exchanges/account/account_test.go

```diff
@@ -103,7 +103,7 @@ func TestHoldings(t *testing.T) {
 		t.Error("error cannot be nil")
 	}
 
-	_, err = GetHoldings("bla", asset.Item("hi"))
+	_, err = GetHoldings("bla", asset.Empty)
 	if err == nil {
 		t.Error("error cannot be nil since an invalid asset type is provided")
```

Comment:
Update with appropriate error check

Reply:
Done in diff PR [HERE](https://github.com/thrasher-corp/gocryptotrader/pull/916/files#diff-f5bc4d5f33a4ae9b6797dce313ae20a48b2e4a44848409ea273c9fd5089b92f2R141)